src/index.html

```diff
@@ -7,7 +7,56 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="body">
+    <div class="container">
+      <div class="stars--0">
+        <div class="stars__star_not_active"></div>
+        <div class="stars__star_not_active"></div>
+        <div class="stars__star_not_active"></div>
+        <div class="stars__star_not_active"></div>
+        <div class="stars__star_not_active"></div>
+      </div>
+      <div class="stars--1">
+        <div class="stars__star_active"></div>
+        <div class="stars__star_not_active"></div>
+        <div class="stars__star_not_active"></div>
+        <div class="stars__star_not_active"></div>
+        <div class="stars__star_not_active"></div>
```

Comment:
all stars at your containers should have only `stars__star` class. 
Use Pseudo Classes to select what stars should be active